test/Frontend/skip-function-bodies.swift

```diff
@@ -174,9 +189,9 @@ public func inlinableNestedLocalTypeFunc() {
     typealias InlinableNestedLocalType = Int
     func takesLocalType(_ x: InlinableNestedLocalType) {
       let ALLNOTYPECHECK_innerLocal2 = 1
-      _blackHole("nested func body inside @inlinable func body taking local type")
-      // CHECK-NONINLINE-ONLY: "nested func body inside @inlinable func body taking local type"
-      // CHECK-ALL-ONLY-NOT: "nested func body inside @inlinable func body taking local type"
+      _blackHole("nested func body inside @inlinable func body taking nested local type")
+      // CHECK-NONINLINE-ONLY: "nested func body inside @inlinable func body taking nested local type"
+      // CHECK-ALL-ONLY-NOT: "nested func body inside @inlinable func body taking nested local type"
```

Comment:
There's another function above this that had the same message. Just added an extra nested so they were different